pom.xml

```diff
@@ -170,17 +170,18 @@
         <xalan.bundle.version>2.7.2_3</xalan.bundle.version>
         <!--END versions that MUST exactly match the versions used by the CXF and Camel features-->
         <!-- Artemis dependencies -->
-        <artemis.version>2.1.0</artemis.version>
+        <artemis.version>2.4.0</artemis.version>
```

Comment:
Don't forget to update the other dependencies that go along with upgrading artemis. I believe proton, qpid-jms, and netty all need to be updated to later versions with the upgrade of artemis to 2.4.

Reply:
Are all the versions in that `features.xml` the only versions I need to make sure to bump? Or is there somewhere else I should be looking as well?

Reply:
fixed